string target_frame
float32 loop_rate
float64 angle_offset
int32 num_waypoints
bool interpolate_xy
bool interpolate_z
bool interpolate_yaw
---
bool success
